app/furniture/marketplace/README.md

```diff
@@ -4,7 +4,9 @@
 
 ### Shopper Purchases Products
 
-<!-- it would be cool if someone could Mermaid this -->
+<!-- Chart created with Mermaid see: docs/checkout_flow_diagram_mermaid.txt -->
+
+![](../../../docs/checkout_flow_diagram.png)
```

Comment:
If you hate how the chart is dark, here's another color option, which IMHO is worse.
![mermaid-diagram-2023-01-21-214228](https://user-images.githubusercontent.com/16140873/213902401-c8fd75e1-c1d4-4103-b5a1-fe9ad9940cf4.png)

Reply:
FYI, Github's Markdown renderer supports Mermaid! You can put the Mermaid code inline in markdown files, so you don't need to compile the png file!

Reply:
weird I tried that and it only displayed the code as code. Not as a chart.

Reply:
<img width="1680" alt="Screen Shot 2023-01-22 at 9 49 03 AM" src="https://user-images.githubusercontent.com/16140873/213931603-7fc3e6a9-8e0c-4a06-a3c5-8f9774dbcd96.png">

Reply:
Did you enclose it in backticks with:


 `````
 ```mermaid
 
 ```
 `````
 
 Per https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/?

Reply:
yes.
<img width="1680" alt="Screen Shot 2023-01-22 at 9 56 27 AM" src="https://user-images.githubusercontent.com/16140873/213931921-34efa5b7-5e7b-4d4f-a8a0-d9d322b33229.png">

Reply:
This is the view from inside your editor; not github.com's UI. You will likely need to install a VS Code Extension for the Markdown previewer to support mermaid directly.

Reply:
Fixed! It [renders](https://github.com/zinc-collective/convene/blob/f6cf6540bd4cca4166b6f86045a7fd3debe682a6/app/furniture/marketplace/README.md)!